src/components/Auth/SignUpForm/SignUpSecondStepForm/index.tsx

```diff
@@ -100,13 +100,13 @@ function SignUpSecondForm() {
   };
```

Comment:
too many logic, move to custom hook

Reply:
What logic to move in a custom hook, because the logic from onSubmit doesn't need a custom hook, and a hook from the library "react-hook-form" is already used for input processing. Maybe I didn't understand you correctly, can you explain what logic you mean?

Reply:
everything that before line 101

Reply:
resolved